CRM/Utils/SQL/Select.php

```diff
@@ -594,6 +594,25 @@ public function toSQL() {
   }
 
   /**
+   * @return CRM_Core_DAO
+   */
```

Comment:
It would be nice to flesh out this docblock by:

* providing a short description
* documenting the parameters with types and descriptions (e.g. what happens when `$i18nRewrite` is `TRUE` vs `FALSE`?